src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -216,7 +216,7 @@ public OperationMethod(Operation operation, Method method)
                 Schema schema = schemaResponse.Schema is ConstantSchema constantSchema ? constantSchema.ValueType : schemaResponse.Schema;
                 TypeReference responseType = BuilderHelpers.CreateType(schema, isNullable: false);
 
-                ObjectSerialization serialization = SerializationBuilder.Build(httpResponse.KnownMediaType, schema, isNullable: false);
+                ObjectSerialization serialization = SerializationBuilder.Build(httpResponse?.KnownMediaType, schema, isNullable: false);
```

Comment:
Can `httpResponse` be null here?

Reply:
Yes, when you don't have any responses in the operation. These are valid cases on the test server.